airbyte-integrations/bases/base-normalization/dbt-project-template-clickhouse/dbt_project.yml

```diff
@@ -24,7 +24,7 @@ macro-paths: ["macros"]
 
 target-path: "../build" # directory which will store compiled SQL files
 log-path: "../logs" # directory which will store DBT logs
-packages-install-path: "/tmp/dbt_modules" # directory which will store external DBT dependencies
+packages-install-path: "/dbt-tmp/dbt_modules" # directory which will store external DBT dependencies
```

Comment:
for simpler naming, it doesn't need to be related to `tmp`, what about going directly with `/dbt_modules` or even `/dbt` (without the dbt_modules)?

Reply:
I like `/dbt` too. It's not really tmp since these are files we want to keep and continually use.